packages/user-interface/src/pages/ThemeSubPage.tsx

```diff
@@ -8,12 +8,14 @@ import { RouterOutletContext } from "../contexts/RouterOutletContext";
 interface Props {
   type: string;
   titleTranslationId?: string;
+  pageHeaderContent?: React.ReactNode;
   children?: React.ReactNode;
 }
 
 const ThemeSubPage = ({
   type,
   titleTranslationId = `pageTitles.${type}`,
+  pageHeaderContent,
```

Comment:
Zou het hier mooier zijn als titleTranslationId null kan zijn dat je de PageHeader niet renderd en dat je bij de implementatie zelf de pageHeader nog toevoegt? Vind de pageHeaderContent var misschien hier helemaal passen bij dit component. En misschien hoef je dan die Paragraph ook helemaal niet weg te halen uit de PageHeader?

Reply:
Goed punt, maar aan de andere kant: als je dat deel van de ThemeSubPage niet meer gebruikt, dan kun je net zo goed de ThemeSubPage helemaal niet gebruiken. Dan heb je ook niet meer het probleem dat de PageGrid in de weg zit als je de PageHeader in de children van de ThemeSubPage stopt.

Reply:
Zoals besproken heb ik de pageHeaderContent prop weggehaald, en lossen we het op in onze eigen custom pagina.